src/elm/Api.elm

```diff
@@ -106,3 +107,27 @@ getExpiryOpts shared symbol toMsg =
                 )
                 |> Http.expectJson toMsg
         }
+
+
+getSupply : Shared -> Eos.Symbol -> (Result Http.Error Eos.Asset -> msg) -> Cmd msg
+getSupply shared symbol toMsg =
+    let
+        query =
+            { code = shared.contracts.token
+            , scope = Eos.symbolToSymbolCodeString symbol
+            , table = "stat"
+            , limit = 1
+            }
+    in
+    Http.post
+        { url = blockchainUrl shared [ "chain", "get_table_rows" ] []
+        , body =
+            Eos.encodeTableQuery query
+                |> Http.jsonBody
+        , expect =
+            Decode.field "rows"
+                (Decode.index 0
+                    (Decode.field "supply" Eos.decodeAsset)
+                )
+                |> Http.expectJson toMsg
+        }
```

Comment:
for some reason my mind thought you were already getting the minBalance from this function 🙃